ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -477,7 +477,7 @@ private void LoadSpreadsheetDocument(SpreadsheetDocument dSpreadsheet)
                                 continue;
                         }
 
-                        if (target != null && source != null)
+                        if (source != null)
```

Comment:
I assume the static analysis tool said that `target` cannot be `null` here. So, for checks like these, I prefer to keep the check, to future-proof it. It's possible that something happens in the future that changes code higher up and then we would need the check. Rather than introducing a possible regression in the future, I'd prefer to keep this check her. It doesn't hurt to keep it.

Reply:
OK, got it. Will restore it.